images/base/Dockerfile

```diff
@@ -121,6 +121,8 @@ COPY [ "entrypoint", "/usr/local/bin/" ]
 # NOTE: this is *only* for documentation, the entrypoint is overridden by the node image
 ENTRYPOINT [ "/usr/local/bin/entrypoint", "/sbin/init" ]
 
-# TODO(bentheelder): deal with systemd MAC address assignment
+# deal with systemd MAC address assignment
```

Comment:
@BenTheElder can you give more context on this issue? I don´t remember related failures

Reply:
it's just a known issue with some virtual devices and systemd that I noted for the future, I'm not sure that this is in fact an issue we've had an problem with yet. so far as far as I can tell it is not.

Reply:
it might be equally valid to just remove this comment

Reply:
we went that route, thanks again!
https://github.com/kubernetes-sigs/kind/pull/590
/close